base_domain_operator/expression.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api
+# pylint: disable=W0402
+from openerp.osv.expression import ExtendedLeaf, expression, is_operator,\
```

Comment:
Ok I'm not sure that understand the point.

I mean, if you see the [pylint result](https://travis-ci.org/OCA/server-tools/jobs/205329573#L483) we don't have a message emitted about use `osv.expression` because we are skip it from https://github.com/OCA/pylint-odoo/pull/40